server/src/main/java/org/elasticsearch/cluster/coordination/InstanceHasMasterHealthIndicatorService.java

```diff
@@ -27,6 +31,7 @@ public class InstanceHasMasterHealthIndicatorService implements HealthIndicatorS
 
     private static final String INSTANCE_HAS_MASTER_GREEN_SUMMARY = "Health coordinating instance has a master node.";
     private static final String INSTANCE_HAS_MASTER_RED_SUMMARY = "Health coordinating instance does not have a master node.";
+    private static final String NO_MASTER_IMPACT = "The cluster cannot create, delete, or rebalance indices, and is likely to be unstable";
```

Comment:
> In addition to "create", I think the cluster won't be able to accept any new writes to existing indices (append/update) either?

Ohh, you're right. Writes fail.

> All the _cat (and some other APIs) will not work.

I didn't realize this, but you're right about that, too.

> For "connected service"

Do we need to call this out separately? They only don't work because the various APIs we're already talking about don't work, right?

> Maybe somehow counterintuitively, search will continue to work when the master is missing. Is it worth pointing this out?

If you don't have a master, you probably ought to drop everything and fix that problem. There's no point in making it sound not quite so bad is there?

Reply:
>> For "connected service"

> Do we need to call this out separately? They only don't work because the various APIs we're already talking about don't work, right?

I personally would prefer to make this a bit more explicitly but I will defer this to @shubhaat .  
The question is when master is not available, all connected services like "Kibana/EnterpriseSearch/Fleet" will stop working.  Should these connected services be listed as part of the `impact area`? and should the description of impact includes a statement on the "connected services"?

Reply:
It makes sense to me that we'd want to tell users that. But we don't usually reference Enterprise Search in Elasticsearch code. That seems like something we'd do at the Cloud level -- it sees that Elasticsearch is broken, so it knows to tell the user that all of the applications it is hosting that are built on Elasticsearch are not working. I'll go ahead and merge this for now, and can open a separate PR about this if the consensus is that we want to start referencing applications built on top of Elasticsearch more inside of Elasticsearch.